tests/ordering/tests.py

```diff
@@ -317,6 +320,20 @@ def test_order_by_fk_attname(self):
             attrgetter("headline")
         )
 
+    def test_order_by_value_expression_with_annotate(self):
+        msg = 'Cannot resolve expression type, unknown output_field'
+        with self.assertRaisesMessage(FieldError, msg):
+            list(Article.objects.annotate(headline2=Value('headline')))
+        self.assertQuerysetEqual(
+            Article.objects.annotate(headline2=Value('headline', output_field=CharField())), [
```

Comment:
Didn't you forget to add an order_by here?

Reply:
I forgot it thanks 👍